### DISCUSSION TITLE
How to use Webdriver.FindElement(By.ID..) or FindElements in codeceptjs project

### DISCUSSION THREAD

**Discussion by Arthanarieaswaran** [codeceptjs/CodeceptJS#3696] [Q&A]
How to use plain webdriver object as similar to basic selenium code.

how to get the Webdriver object in the codeceptjs project.

I would like to use

Element ele = webdriver.findElement(By.id('1234')).

is this possible in codecept project?

**Comment by kobenguyent** [comment]
You could try this https://codecept.io/helpers/WebDriver/#usewebdriverto

**Comment by Arthanarieaswaran** [comment]
I am using CodeCeptjs BDD structure, please let me know how to get this **browser** object in the codeceptjs stepdefinition.js file

<img width="828" alt="image" src="https://github.com/codeceptjs/CodeceptJS/assets/6602970/4bd7a2f9-d9df-418c-bc62-60193efde81b">

**Reply by Arthanarieaswaran** [reply]
HI @kobenguyent, i have unmarked, because i have followup question on this, i dont want to open one more for this followup question
